### DISCUSSION TITLE
Are shadows under comboboxes intentionally disabled?

### DISCUSSION THREAD

**Discussion by Pololot64** [bdlukaa/fluent_ui#896] [Q&A]
I was just wondering. It is not very important, but contrast does suffer.

**Comment by bdlukaa** [comment]
Do you mean the combobox button or the combobox flyout?

**Comment by Pololot64** [comment]
The Flyout. ![image](https://github.com/bdlukaa/fluent_ui/assets/31389383/801138fa-b09b-44f1-9898-198ec6f0f6f6) @bdlukaa

**Comment by Pololot64** [comment]
I believe the shadows are just cut off by a ClipRRect.

**Comment by Pololot64** [comment]
Awesome :) The change works well. Thank you.

**Comment by Pololot64** [comment]
@bdlukaa In addition, I believe the same issue also applies to datepickers.
